tests/unit/findIncludeTags.js

```diff
@@ -8,7 +8,7 @@ define([
 		bdd.it('should return an array if include tags are found', function () {
 			var file = fs.readFileSync('tests/data/docs/has_include_tags.md').toString();
 			var findHeadingTags = markdownInclude.findIncludeTags(file);
-			assert.equal(findHeadingTags.length, 2, 'Heading tags length match');
+			assert.equal(findHeadingTags.length, 3, 'Heading tags length match');
```

Comment:
note I was unable to test these changes, when I run `npm test` I get the following error:

```
% npm run test 

> markdown-include-etn@0.4.4 test
> node_modules/.bin/intern-client config=tests/intern.js

internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module 'dojo/dojo'
Require stack:
- /repos/markdown-include/node_modules/@theintern/dojo/dojo.js
- /repos/markdown-include/node_modules/intern/client.js
- /repos/markdown-include/node_modules/intern/bin/intern-client.js
```